### PR TITLE
Add BLIK icon

### PR DESCRIPTION
**Pull request by patryk-smc**
Brand logo: https://polskistandardplatnosci.pl/en/content-to-download/ (they don't have guidelines unfortunately)

** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

### CODE REVIEW

File: app/assets/images/payment_icons/blik.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32z" fill="url(#paint0_linear)"/><path d="M30.343 17.155l-2.785-3.639 2.563-3.236h-2.185l-2.456 3.138V6.78h-1.848v10.375h1.848v-3.532l2.456 3.532h2.407zM18.613 6.78h-1.848v10.366h1.848V6.78zm3.433 3.508h-1.848v6.867h1.848v-6.867z" fill="#fff"/><path d="M13.849 9.573a1.651 1.651 0 100-3.302 1.651 1.651 0 000 3.302z" fill="url(#paint1_radial)"/><path d="M12.041 10.206c-.574 0-1.138.144-1.642.419V6.82H8.534v6.9a3.516 3.516 0 103.507-3.515zm0 5.175a1.643 1.643 0 110-3.286 1.643 1.643 0 010 3.286z" fill="#fff"/><defs><radialGradient id="paint1_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(12.51 6.18) scale(5.41297)"><stop stop-color="red"/><stop offset=".49" stop-color="#E83E49"/><stop offset="1" stop-color="#F0F"/></radialGradient><linearGradient id="paint0_linear" x1="19" y1="4.977" x2="19" y2="55.605" gradientUnits="userSpaceOnUse"><stop/><stop offset=".732" stop-color="#fff"/><stop offset="1" stop-color="#fff"/></linearGradient></defs></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/blik.svg:1]: According to the error log, can you try and rename your `id="paint1_radial"` to `id="pi-blik-paint1_radial"` as well as `id="paint0_linear"`?

Comment (patryk-smc) [reply app/assets/images/payment_icons/blik.svg]: @tauthomas01 Done!

**Comment by tauthomas01** [comment]
The icon looks good.
I'll have to investigate why the test failed.

**Comment by tauthomas01** [comment]
Hi @patryk-smc ,

Can you try and rebase your branch?
For some reasons, other PR have successful tests so I really don't know what's the issue.

**Comment by patryk-smc** [comment]
@tauthomas01 Should be good now

**Review by tauthomas01 — APPROVED** [review]
It works well now 👍

**Pull request merged**